autode/bracket/base.py

```diff
@@ -68,11 +68,13 @@ def _name(self) -> str:
     @property
     def ts_guess(self) -> Optional["Species"]:
         """Get the TS guess from image-pair"""
+        assert self.imgpair is not None, "Must have an image pair for TS guess"
```

Comment:
Are these lines required for mypy? If I understand correctly, mypy checks the types of everything match, sort of like C/C++?